src/routes/notification/template/index.js

```diff
@@ -0,0 +1,9 @@
+import express from 'express';
+import TemplateController from '../../../controllers/TemplateController';
+
+const router = express.Router();
+
+router.get('/', TemplateController.getTemplates);
+router.get('/:id', TemplateController.getTemplate);
+
+export default router;
```

Comment:
These routes need to be registered in the src/makeroutes.js file otherwise they are not accessible.

Reply:
I checked it, these routes works.